build/.mxchip-ci.yml

```diff
@@ -30,13 +30,13 @@ jobs:
 - job: Execute
   dependsOn: Cross_Compile
   pool:
-    name: $(AGENTPOOL)
+    name: $(MXCHIPAGENTPOOL)
   steps:
   - task: DownloadBuildArtifacts@0
     displayName: 'Download Build Artifacts'
     inputs:
       artifactName: 'drop'
 
   - script: |
-      cp $(System.ArtifactsDirectory)/drop/mxchip.bin /media/newt/AZ3166
+      echo $(NEWTIN) | sudo -S cp $(System.ArtifactsDirectory)/drop/mxchip.bin /media/newt/AZ3166
```

Comment:
for a multiline script you can just add the `sudo -S` line at the same indentation as echo. Makes for better looking code.

Also what is NEWTIN?

Reply:
It's not multiline code, it's a pipe.

Reply:
Printing again here, not a multiline script, it's a pipe.